src/Configuration/NodeDefinition/TablesNode.php

```diff
@@ -31,7 +37,6 @@ public function create(): NodeDefinition
                 }
                 return $v;
             })->end()
-            ->ignoreExtraKeys(true)
```

Comment:
Tohle je tady taková propašovaná změna :) Ale dává to smysl - až se bude implementovat ten config do extraktorů tak je stejně potřeba to projít (mmch na to je ta tabulka někde co který db-ex umí za věci v configu).